### PR TITLE
Added user and post classes, added newsfeed activity

### PR DESCRIPTION
**Pull request by JustCourt**

### CODE REVIEW

File: app/src/main/java/com/veephealthoutloud/healthoutloud/Classes/User.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.List;
+
+/**
+ * Class representing a user.
+ */
+
+public class User implements IUser {
+
+    public String _email;
+    public List<IPost> _posts;
```
Comment (chouvinc) [inline app/src/main/java/com/veephealthoutloud/healthoutloud/Classes/User.java]: make these variables private

File: app/src/main/java/com/veephealthoutloud/healthoutloud/NewsfeedActivity.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        PostAdapter postAdapter = new PostAdapter(this, postMessages);
+        postListView.setAdapter(postAdapter);
+    }
+
+    private ArrayList<String> GetPosts(){
+        ArrayList<String> list = new ArrayList<>();
+        list.add("post1");
+        list.add("post2");
+        list.add("post3");
+        return list;
```
Comment (chouvinc) [inline app/src/main/java/com/veephealthoutloud/healthoutloud/NewsfeedActivity.java]: Can you leave a comment here saying it's going to be replaced by calling an API from the backend (do something like // BACKEND NEEDED so they can just ctrl-f)

**Review by chouvinc — CHANGES_REQUESTED** [review]
Just 1 quick fix and 1 comment needed

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -16,8 +16,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".NewsfeedActivity"
+            android:label="@string/title_activity_newsfeed"
+            android:theme="@style/AppTheme.NoActionBar" />
         <activity android:name=".Progress">
-            <intent-filter></intent-filter>
```
Comment (JustCourt) [inline app/src/main/AndroidManifest.xml:20]: @yusufelgharib  i had to remove the intent-filter tags since it was causing the project to not build

**Review by chouvinc — APPROVED** [review]
Looks good!

**Review by yusufelgharib — APPROVED** [review]

**Pull request merged**